main.py

```diff
@@ -104,47 +119,55 @@ def main():
             print("\nWould you like to see the travel route in Google Maps?(y/n)")
             question = input()
             if question.lower() == "y":
-                webbrowser.open(construct_google_map_path(travel_df))
+                webbrowser.open(
+                    construct_google_map_path(
+                        travel_df, geocodes_df, start_lat, start_lon
+                    )
+                )
                 exit()
             else:
                 exit()
 
 
-def display_travel_route(travel_df, distance_to_start, total_distance):
+def generate_travel_route(
+    travel_df,
+    geocodes_df,
+    breweries_df,
+    start_lat,
+    start_lon,
+    distance_to_start,
+    total_distance,
+):
     """
     Format a detailed list of breweries visited on the travel route
     """
     s = ""
     number_of_breweries = travel_df["brewery_id"].count()
     s += f"Found {number_of_breweries} beer factories:\n"
     s += f"-> HOME: {start_lat} {start_lon}\n"
-    for row in travel_df.index:
-        br_id = travel_df.loc[row]["brewery_id"]
-        lat, lon = geocodes_df.loc[br_id, "latitude":"longitude"]
-        name = breweries_df.loc[br_id]["name"]
-        distance = travel_df.loc[row]["distance"]
+    for row in travel_df.itertuples():
```

Comment:
It would be nicer if you could do the following
```py
for lat, lon, name, distance in travel_df[['lat', 'lon', 'name', 'distance']].values:
     # do whatever you want
```
The same comment below

Reply:
`for lat, lon, name, distance in travel_df.itertuples():`
is also viable
or 
```py
for travel in travel_df.itertuples()
     travel.lat, travel.lon, ....
```

Reply:
Fixed